src/main/java/seedu/fast/logic/parser/AddCommandParser.java

```diff
@@ -49,8 +50,9 @@ public AddCommand parse(String args) throws ParseException {
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
         Appointment appointment = new Appointment(Appointment.NO_APPOINTMENT, Appointment.NO_TIME,
                 Appointment.NO_VENUE);
+        AppointmentCount count = new AppointmentCount("0");
```

Comment:
Maybe to avoid magic literals (even though its quite obvious), you can create a constant INITIAL_COUNT and put it inside the constructor.